src/snowflake/snowpark/lineage.py

```diff
@@ -372,7 +372,10 @@ def _get_user_entity(self, graph_entity) -> str:
         )
 
         # TODO: Remove this workaround once version 8.18 is deployed.
-        if graph_entity.get(_ObjectField.REFINED_DOMAIN) == _SnowflakeDomain.VIEW:
+        if (
```

Comment:
do we need test to cover this branching logic change?

Reply:
Added it.